README.md

```diff
@@ -22,7 +21,7 @@ Or, in graph form [^fn1]:
 %%{ init: { 'flowchart': { 'curve': 'bumpX' } } }%%
 graph LR;
 linkStyle default opacity:0.5
-  design_system_react_native(["@metamask/design-system-react-native"]);
+  design_system_react_native(["@metamask/storybook-react-native"]);
```

Comment:
We should keep the reference to `packages/design-system-react-native` because it's an exported module from the monorepo. However, we can remove the reference to `react-native storybook` as it's not a package. Revert the changes to this file

``` git checkout main -- README.md```